cmd/ttn-lw-cli/commands/organizations.go

```diff
@@ -141,7 +145,11 @@ var (
 		Short:   "Create an organization",
 		RunE: asBulk(func(cmd *cobra.Command, args []string) (err error) {
 			orgID := getOrganizationID(cmd.Flags(), args)
-			collaborator := getUserID(cmd.Flags(), nil).OrganizationOrUserIdentifiers()
+			usrID, err := getUserID(cmd.Flags(), nil)
+			if err != nil {
+				return err
+			}
+			collaborator := usrID.OrganizationOrUserIdentifiers()
 			if collaborator == nil {
```

Comment:
Can remove this check and just use `usrID.OrganizationOrUserIdentifiers()` directly like you do above in list.